### PR TITLE
ST-483: Manage wish list

### PR DESCRIPTION
**Pull request by ilyawzrd**
Description:

--

QA-test: ST-486

Demo-test: ST-487

Download artifact URL: https://vc3prerelease.blob.core.windows.net/packages/client-app-0.0.4-pr-103-468a6c81.zip

### CODE REVIEW

File: client-app/router.ts [hunk 1]
```diff
@@ -47,7 +48,7 @@ const routes: RouteRecordRaw[] = [
       { path: "orders", name: "Orders", component: Orders },
       { path: "order-details/:id", name: "OrderDetails", component: OrderDetails },
       { path: "lists", name: "Lists", component: Lists },
-      { path: "lists/:listId", name: "ListDetails", component: {}, props: true }, // TODO: implement in another user story
+      { path: "lists/:listId", name: "ListDetails", component: ListDetails, props: true }, // TODO: implement in another user story
```
Comment (Flanker72) [inline client-app/router.ts:51]: Remove TODO - it is implemented right here)

Comment (ilyawzrd) [reply client-app/router.ts]: done, text blended into my vscode background)

File: client-app/pages/account/list-details.vue [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+            <div v-if="!isMobile" class="flex flex-col bg-white rounded border shadow-sm">
+              <WishlistProductItem
+                v-for="item in listItems"
+                :key="item.id"
+                :line-item="item"
+                class="even:bg-gray-50"
+                @remove="openDeleteProductDialog(item)"
+              ></WishlistProductItem>
+              <div class="flex p-5" v-if="pages > 1">
+                <VcPagination v-model:page="page" :pages="pages" @update:page="page = $event"></VcPagination>
```
[the remark this answers is not in the document]
Comment (ilyawzrd) [reply client-app/pages/account/list-details.vue:49]: done

File: client-app/pages/account/list-details.vue [hunk 3]
```diff
@@ -0,0 +1,198 @@
...
+          </div>
+
+          <!-- Mobile footer block -->
+          <div v-if="isMobile" class="flex flex-col space-y-4">
+            <VcPagination
+              v-if="pages > 1"
+              v-model:page="page"
+              :pages="pages"
+              class="mb-3 lg:mb-0"
+              @update:page="page = $event"
```
[the remark this answers is not in the document]
Comment (ilyawzrd) [reply client-app/pages/account/list-details.vue:97]: done

**Review by artmakarov — APPROVED** [review]

File: client-app/shared/wishlists/components/wishlist-product-item.vue [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+    <div class="relative shrink-0 w-full flex-1 flex space-x-3 items-center">
+      <div class="border border-gray-100 w-16 h-16 shrink-0">
+        <VcImage
+          :src="lineItem.product?.imgSrc"
+          :alt="lineItem.name"
+          class="w-full h-full object-cover object-center"
+        />
+      </div>
+      <router-link
+        :to="`/${SeoUrl.Product}/${lineItem.product?.id}`"
```
[the remark this answers is not in the document]
Comment (ilyawzrd) [reply client-app/shared/wishlists/components/wishlist-product-item.vue]: done

**Pull request merged**